docs/source/tutorial/tutorial-introduction.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Step 0: Introduction"
```

Comment:
I think "Step" is redundant here, especially for the left sidebar. In the [fullstack tutorial](https://www.apollographql.com/docs/tutorial/introduction/) we just use `0.`, `1.`, etc. as the prefixes. Should we do that here as well?

Reply:
Ah! Yes. Will address